marketplace_builder/views/pages/release-notes/25-FEBRUARY-2019.liquid

```diff
@@ -0,0 +1,24 @@
+---
+converter: markdown
+metadata:
+  title: February 25, 2019  &ndash;  Release Notes
+  date: February 25, 2019
+  description: context.device key, user session timeout
+  hide_sidebar: true
+  exclude_contributors: true
+slug: release-notes/25-FEBRUARY-2019
+searchable: true
+---
+
+{{ page.metadata.description }}
+
+## What's new
+
+* **Added `context.device` key**: It includes information about device type, like desktop, smartphone, etc. Should save you some work. It has been added in a lazy way, so it will not cause performance overhead if you don’t use it.
+* **Search by ID for `admin_*` queries**: Added the possibility to search by ID for `admin_*` queries. Also, added metadata to all `admin_*` resources (possible to set them via GraphQL).
+* **User session timeout**: You can now control when a user session timeouts via the Partner Portal.
```

Comment:
```suggestion
* **User session timeout**: You can now control when a user session timeouts via the Partner Portal.
```

I dont know how to write it, but, you cant control when it timeouts, you can control after what time it timeouts from the start of it. But maybe im knitpicking the logic in here ;)

Reply:
Also we can link to: `/tutorials/partner-portal/updating-instance-configuration` to help people get started with it

Reply:
Thank you! I rephrased the sentence to make it more precise, and added link to tutorial.